sass/_components.scss

```diff
@@ -296,10 +303,22 @@
 // .no-block-title - No block title (black/white heading)
 //
 // markup:
-// <div class="pullout-block pullout-block__text {$modifiers}">
+// <div class="pullout-block pullout-block__text no-arrows {$modifiers}">
```

Comment:
Bit misleading having a no-arrows class on here and then adding arrows to the list.

Reply:
This instance of the "no-arrows" class is to remove the single arrow added to the entire block on mobile. I can explicitly override that style for this component if you'd prefer.